docs/source/extend_modules/data_source.rst

```diff
@@ -1,17 +1,73 @@
 Add new Data Source
 =======================
 
-VISSL supports data loading from :code:`disk_folder` as the default data source. If your dataset lives in a custom data storage solution instead of :code:`disk_folder`, you can extend VISSL to work with your data storage in several different ways:
+VISSL supports data loading from :code:`disk_folder` or :code:`disk_filelist` as the default data source.
```

Comment:
let's expand this to include also `torchvision_dataset` :)

Reply:
Indeed, it's a valid data source now :)

The documentation should be good following commit 6813389.